config/locales/de.yml

```diff
@@ -3,5 +3,8 @@ de:
     errors:
       messages:
         content_type: "ist kein gültiges Dateiformat"
+        content_type_with_filename: "%{filename} ist kein gültiges Dateiformat"
```

Comment:
Do not change the default i18n messages.
It is sufficient to verify by test that the argument is optionally passed.